cleanlab/multilabel_classification/filter.py

```diff
@@ -241,27 +265,39 @@ def find_multilabel_issues_per_class(
             raise ValueError("confident_joint should be of shape (num_classes, 2, 2)")
     for class_num, (label, pred_prob_for_class) in enumerate(zip(y_one.T, pred_probs.T)):
         pred_probs_binary = stack_complement(pred_prob_for_class)
```

Comment:
(can't comment on the exact line so will comment here)

I believe the `confident_joint` computation (starts a few line above) can also be skipped if `low_memory=True`